### ISSUE TITLE
Incorrect Multipolygon Validation

### ISSUE THREAD

**Issue by Den3** [peterstace/simplefeatures#215]
example WKT
```
MULTIPOLYGON(((151.207628272004 -33.8551079129544,151.207630907003 -33.8551143401547,151.207625421003 -33.8551184904543,151.207573878005 -33.8551573762498,151.207347896015 -33.8553278442305,151.207292893029 -33.8552800124247,151.207232037058 -33.8551364722171,151.207225278061 -33.8551203729162,151.207382497044 -33.8550739215306,151.207589287023 -33.8550128320495,151.207620446008 -33.8550888286534,151.207628272004 -33.8551079129544)),((151.205319518151 -33.8564836954525,151.205286826154 -33.8564970876495,151.205264569157 -33.8564954113474,151.205228892175 -33.8564059687429,151.205307165166 -33.85638322715,151.205294679172 -33.8563536728485,151.205324780169 -33.8563445027512,151.205389159147 -33.8564347700585,151.205382786146 -33.8564524200581,151.205319518151 -33.8564836954525)),((151.207630907003 -33.8551143401547,151.207680998022 -33.854918118657,151.207200663072 -33.8550625373132,151.20709743412 -33.8548190487003,151.206826488248 -33.8541726291663,151.206642090334 -33.8537377416432,151.206639831334 -33.853737625143,151.206639285335 -33.8537365494429,151.206638702335 -33.8537354917428,151.206638083335 -33.8537344518428,151.206637438335 -33.8537334389427,151.206636757336 -33.8537324349426,151.206636040336 -33.8537314579425,151.206635305336 -33.8537305077424,151.206634534336 -33.8537295754424,151.206633727337 -33.8537286611423,151.206632902337 -33.8537277916422,151.206632051337 -33.8537269311421,151.206631163337 -33.853726106442,151.206630258337 -33.8537253086419,151.206629326338 -33.8537245377418,151.206628366338 -33.8537237937417,151.206627398338 -33.8537230855416,151.206626394338 -33.8537224132415,151.206625372339 -33.8537217588414,151.206624342339 -33.8537211493413,151.206623284339 -33.8537205666412,151.206622199339 -33.8537200109411,151.20662111534 -33.853719499941,151.20662000334 -33.8537190248408,151.20661887434 -33.8537185766407,151.20661773534 -33.8537181643406,151.20661658834 -33.8537177968405,151.206615422341 -33.8537174561404,151.206614257341 -33.8537171514403,151.206613083341 -33.8537168914401,151.206611891341 -33.8537166583401,151.206610698342 -33.8537164701399,151.206609497342 -33.8537163177398,151.206608287342 -33.8537162101397,151.206607086342 -33.8537161295396,151.206605876342 -33.8537160936395,151.206604666342 -33.8537160936393,151.206603456343 -33.8537161295392,151.206602254343 -33.8537162012391,151.206601053343 -33.853716317739,151.206599852343 -33.8537164701389,151.206598651343 -33.8537166494388,151.206597468343 -33.8537168735387,151.206596284344 -33.8537171424386,151.206595119344 -33.8537174382385,151.206593963344 -33.8537177788384,151.206592806344 -33.8537181464383,151.206591668344 -33.8537185587382,151.206590547344 -33.853718997938,151.20643799836 -33.8537627422241,151.20625118938 -33.853817592707,151.20625012238 -33.8538177451069,151.20624905638 -33.8538179334069,151.20624800738 -33.8538181664067,151.20624695838 -33.8538184354067,151.20624592738 -33.8538187491066,151.206244905381 -33.8538190897065,151.206243892381 -33.8538194752064,151.206242897381 -33.8538198965063,151.206241920381 -33.8538203537062,151.206240952381 -33.8538208467061,151.206240011381 -33.853821375506,151.206239097381 -33.853821931306,151.206238191381 -33.8538225319059,151.206237313381 -33.8538231594058,151.206236461381 -33.8538238138057,151.206235628381 -33.8538245040057,151.206234821381 -33.8538252211056,151.206234050381 -33.8538259651055,151.206233297381 -33.8538267450054,151.206232580381 -33.8538275517054,151.206231890381 -33.8538283764054,151.206231226381 -33.8538292370053,151.206230608381 -33.8538301154052,151.206230016381 -33.8538310118052,151.206229451381 -33.8538319441052,151.206228923381 -33.8538328853051,151.206228438381 -33.8538338444051,151.206227981381 -33.8538348215051,151.206227560381 -33.853835816505,151.206227175381 -33.853836820505,151.206226825381 -33.853837842405,151.20622652038 -33.853838882205,151.20622625138 -33.853839922005,151.20622601838 -33.8538409798049,151.20622583038 -33.8538420465049,151.20622567838 -33.8538431132049,151.20622556138 -33.853844188905,151.20622548938 -33.853845264605,151.20622545338 -33.853846340205,151.206225462379 -33.853847415905,151.206225507379 -33.853848491605,151.206225597379 -33.853849567305,151.206225722379 -33.853850634005,151.206225884379 -33.8538517007051,151.206226081379 -33.8538527674051,151.206226323378 -33.8538538162051,151.206226601378 -33.8538548650052,151.206226924378 -33.8538558958052,151.206225122378 -33.8538580382051,151.206596159204 -33.8547421735516,151.20679553611 -33.8552191468767,151.206795993105 -33.8552485756771,151.206799005079 -33.8554419466798,151.206584308087 -33.8556098867616,151.206345947096 -33.8557932180412,151.206243265124 -33.8556996520303,151.205587388433 -33.854135087848,151.205173880476 -33.8542582440103,151.205910020129 -33.8560130596026,151.205873339116 -33.8561523240009,151.20577423412 -33.8562296293925,151.205752030126 -33.8562095500901,151.205543662134 -33.8563694763723,151.205441840141 -33.8564223011633,151.205422505145 -33.8564182404615,151.20537643016 -33.8563539238562,151.205543492147 -33.8562706933711,151.205487897166 -33.8561929577648,151.205497175165 -33.8561883412656,151.205461731177 -33.8561387883616,151.205459750177 -33.8561393620614,151.204833545472 -33.8546496202829,151.204643365492 -33.8547052775655,151.204618669503 -33.8546464828624,151.204630358504 -33.8546273626633,151.204823613491 -33.8545181365803,151.204798280499 -33.8544869419775,151.204560296515 -33.8546214464566,151.204599863496 -33.8547156399615,151.204608172496 -33.8547132107623,151.204667308468 -33.8548538824697,151.204627579472 -33.8548656162661,151.204625957473 -33.8548616542659,151.204638363471 -33.854857916267,151.204548400514 -33.8546436591557,151.204527568518 -33.8546351703537,151.204175185555 -33.8547385160215,151.204163245554 -33.8547593931206,151.204182724545 -33.854806194023,151.204161461547 -33.8548122537211,151.204164670545 -33.8548199089215,151.203911465572 -33.8548931983984,151.203926229565 -33.8549283462002,151.203952592562 -33.8549206372027,151.204034585524 -33.8551157740129,151.20407028052 -33.8551053310162,151.204632025256 -33.8564423893866,151.204643159254 -33.8564422817877,151.204646359246 -33.8564994271887,151.204675528242 -33.8564991402915,151.204676836229 -33.8565916843928,151.2048767512 -33.8565897213118,151.204875873209 -33.8565276906109,151.204880408209 -33.8565235134113,151.204901796207 -33.8565176061132,151.204922620204 -33.8565118512151,151.20495257719 -33.8565840113189,151.204943999189 -33.8565986853183,151.204880301198 -33.8566069411123,151.204708148221 -33.8566188004961,151.20464806223 -33.8566194279904,151.204187788296 -33.8566242505467,151.204115046331 -33.8564396732375,151.204091784342 -33.8563840875345,151.204262754334 -33.8562659871493,151.203786355558 -33.8551279649895,151.20360903857 -33.8552309073739,151.203561870592 -33.855118543868,151.203144829636 -33.8552393245299,151.203346089541 -33.8557214521552,151.20362321041 -33.85638259949,151.203527645405 -33.8565201696826,151.203492990409 -33.8565340458795,151.203434456421 -33.8565047785736,151.203314105447 -33.8564437069614,151.203236531443 -33.8565508354554,151.202796516536 -33.8563379140108,151.202677519692 -33.8553214075866,151.202666852706 -33.8552334709844,151.202575007826 -33.8544543935658,151.208662051201 -33.8525697173203,151.208387942101 -33.8535954117073,151.208364340093 -33.8536786064061,151.208363201092 -33.8536826582061,151.208062297984 -33.8547940840916,151.207724660999 -33.8550434888627,151.207630907003 -33.8551143401547)))
```

This is working in [wkt reader](https://arthur-e.github.io/Wicket/sandbox-gmaps3.html). But in simplefeatures, it complains about `polygons must not be nested`.

Example code
```
package main

import "github.com/peterstace/simplefeatures/geom"

func main() {
	_, err := geom.UnmarshalWKT(`MULTIPOLYGON(((151.207628272004 -33.8551079129544,151.207630907003 -33.8551143401547,151.207625421003 -33.8551184904543,151.207573878005 -33.8551573762498,151.207347896015 -33.8553278442305,151.207292893029 -33.8552800124247,151.207232037058 -33.8551364722171,151.207225278061 -33.8551203729162,151.207382497044 -33.8550739215306,151.207589287023 -33.8550128320495,151.207620446008 -33.8550888286534,151.207628272004 -33.8551079129544)),((151.205319518151 -33.8564836954525,151.205286826154 -33.8564970876495,151.205264569157 -33.8564954113474,151.205228892175 -33.8564059687429,151.205307165166 -33.85638322715,151.205294679172 -33.8563536728485,151.205324780169 -33.8563445027512,151.205389159147 -33.8564347700585,151.205382786146 -33.8564524200581,151.205319518151 -33.8564836954525)),((151.207630907003 -33.8551143401547,151.207680998022 -33.854918118657,151.207200663072 -33.8550625373132,151.20709743412 -33.8548190487003,151.206826488248 -33.8541726291663,151.206642090334 -33.8537377416432,151.206639831334 -33.853737625143,151.206639285335 -33.8537365494429,151.206638702335 -33.8537354917428,151.206638083335 -33.8537344518428,151.206637438335 -33.8537334389427,151.206636757336 -33.8537324349426,151.206636040336 -33.8537314579425,151.206635305336 -33.8537305077424,151.206634534336 -33.8537295754424,151.206633727337 -33.8537286611423,151.206632902337 -33.8537277916422,151.206632051337 -33.8537269311421,151.206631163337 -33.853726106442,151.206630258337 -33.8537253086419,151.206629326338 -33.8537245377418,151.206628366338 -33.8537237937417,151.206627398338 -33.8537230855416,151.206626394338 -33.8537224132415,151.206625372339 -33.8537217588414,151.206624342339 -33.8537211493413,151.206623284339 -33.8537205666412,151.206622199339 -33.8537200109411,151.20662111534 -33.853719499941,151.20662000334 -33.8537190248408,151.20661887434 -33.8537185766407,151.20661773534 -33.8537181643406,151.20661658834 -33.8537177968405,151.206615422341 -33.8537174561404,151.206614257341 -33.8537171514403,151.206613083341 -33.8537168914401,151.206611891341 -33.8537166583401,151.206610698342 -33.8537164701399,151.206609497342 -33.8537163177398,151.206608287342 -33.8537162101397,151.206607086342 -33.8537161295396,151.206605876342 -33.8537160936395,151.206604666342 -33.8537160936393,151.206603456343 -33.8537161295392,151.206602254343 -33.8537162012391,151.206601053343 -33.853716317739,151.206599852343 -33.8537164701389,151.206598651343 -33.8537166494388,151.206597468343 -33.8537168735387,151.206596284344 -33.8537171424386,151.206595119344 -33.8537174382385,151.206593963344 -33.8537177788384,151.206592806344 -33.8537181464383,151.206591668344 -33.8537185587382,151.206590547344 -33.853718997938,151.20643799836 -33.8537627422241,151.20625118938 -33.853817592707,151.20625012238 -33.8538177451069,151.20624905638 -33.8538179334069,151.20624800738 -33.8538181664067,151.20624695838 -33.8538184354067,151.20624592738 -33.8538187491066,151.206244905381 -33.8538190897065,151.206243892381 -33.8538194752064,151.206242897381 -33.8538198965063,151.206241920381 -33.8538203537062,151.206240952381 -33.8538208467061,151.206240011381 -33.853821375506,151.206239097381 -33.853821931306,151.206238191381 -33.8538225319059,151.206237313381 -33.8538231594058,151.206236461381 -33.8538238138057,151.206235628381 -33.8538245040057,151.206234821381 -33.8538252211056,151.206234050381 -33.8538259651055,151.206233297381 -33.8538267450054,151.206232580381 -33.8538275517054,151.206231890381 -33.8538283764054,151.206231226381 -33.8538292370053,151.206230608381 -33.8538301154052,151.206230016381 -33.8538310118052,151.206229451381 -33.8538319441052,151.206228923381 -33.8538328853051,151.206228438381 -33.8538338444051,151.206227981381 -33.8538348215051,151.206227560381 -33.853835816505,151.206227175381 -33.853836820505,151.206226825381 -33.853837842405,151.20622652038 -33.853838882205,151.20622625138 -33.853839922005,151.20622601838 -33.8538409798049,151.20622583038 -33.8538420465049,151.20622567838 -33.8538431132049,151.20622556138 -33.853844188905,151.20622548938 -33.853845264605,151.20622545338 -33.853846340205,151.206225462379 -33.853847415905,151.206225507379 -33.853848491605,151.206225597379 -33.853849567305,151.206225722379 -33.853850634005,151.206225884379 -33.8538517007051,151.206226081379 -33.8538527674051,151.206226323378 -33.8538538162051,151.206226601378 -33.8538548650052,151.206226924378 -33.8538558958052,151.206225122378 -33.8538580382051,151.206596159204 -33.8547421735516,151.20679553611 -33.8552191468767,151.206795993105 -33.8552485756771,151.206799005079 -33.8554419466798,151.206584308087 -33.8556098867616,151.206345947096 -33.8557932180412,151.206243265124 -33.8556996520303,151.205587388433 -33.854135087848,151.205173880476 -33.8542582440103,151.205910020129 -33.8560130596026,151.205873339116 -33.8561523240009,151.20577423412 -33.8562296293925,151.205752030126 -33.8562095500901,151.205543662134 -33.8563694763723,151.205441840141 -33.8564223011633,151.205422505145 -33.8564182404615,151.20537643016 -33.8563539238562,151.205543492147 -33.8562706933711,151.205487897166 -33.8561929577648,151.205497175165 -33.8561883412656,151.205461731177 -33.8561387883616,151.205459750177 -33.8561393620614,151.204833545472 -33.8546496202829,151.204643365492 -33.8547052775655,151.204618669503 -33.8546464828624,151.204630358504 -33.8546273626633,151.204823613491 -33.8545181365803,151.204798280499 -33.8544869419775,151.204560296515 -33.8546214464566,151.204599863496 -33.8547156399615,151.204608172496 -33.8547132107623,151.204667308468 -33.8548538824697,151.204627579472 -33.8548656162661,151.204625957473 -33.8548616542659,151.204638363471 -33.854857916267,151.204548400514 -33.8546436591557,151.204527568518 -33.8546351703537,151.204175185555 -33.8547385160215,151.204163245554 -33.8547593931206,151.204182724545 -33.854806194023,151.204161461547 -33.8548122537211,151.204164670545 -33.8548199089215,151.203911465572 -33.8548931983984,151.203926229565 -33.8549283462002,151.203952592562 -33.8549206372027,151.204034585524 -33.8551157740129,151.20407028052 -33.8551053310162,151.204632025256 -33.8564423893866,151.204643159254 -33.8564422817877,151.204646359246 -33.8564994271887,151.204675528242 -33.8564991402915,151.204676836229 -33.8565916843928,151.2048767512 -33.8565897213118,151.204875873209 -33.8565276906109,151.204880408209 -33.8565235134113,151.204901796207 -33.8565176061132,151.204922620204 -33.8565118512151,151.20495257719 -33.8565840113189,151.204943999189 -33.8565986853183,151.204880301198 -33.8566069411123,151.204708148221 -33.8566188004961,151.20464806223 -33.8566194279904,151.204187788296 -33.8566242505467,151.204115046331 -33.8564396732375,151.204091784342 -33.8563840875345,151.204262754334 -33.8562659871493,151.203786355558 -33.8551279649895,151.20360903857 -33.8552309073739,151.203561870592 -33.855118543868,151.203144829636 -33.8552393245299,151.203346089541 -33.8557214521552,151.20362321041 -33.85638259949,151.203527645405 -33.8565201696826,151.203492990409 -33.8565340458795,151.203434456421 -33.8565047785736,151.203314105447 -33.8564437069614,151.203236531443 -33.8565508354554,151.202796516536 -33.8563379140108,151.202677519692 -33.8553214075866,151.202666852706 -33.8552334709844,151.202575007826 -33.8544543935658,151.208662051201 -33.8525697173203,151.208387942101 -33.8535954117073,151.208364340093 -33.8536786064061,151.208363201092 -33.8536826582061,151.208062297984 -33.8547940840916,151.207724660999 -33.8550434888627,151.207630907003 -33.8551143401547)))`)
	if err != nil {
		panic(err)
	}
}
```

**Comment by peterstace** [comment]
Confirmed that this is definitely a bug. PostGIS reports this as valid. I'll make a fix over the weekend.

GeoJSON below (which is a bit easier to copy/paste):

```
{
  "type": "MultiPolygon",
  "coordinates": [
    [
      [
        [
          151.207628272004,
          -33.8551079129544
        ],
        [
          151.207630907003,
          -33.8551143401547
        ],
        [
          151.207625421003,
          -33.8551184904543
        ],
        [
          151.207573878005,
          -33.8551573762498
        ],
        [
          151.207347896015,
          -33.8553278442305
        ],
        [
          151.207292893029,
          -33.8552800124247
        ],
        [
          151.207232037058,
          -33.8551364722171
        ],
        [
          151.207225278061,
          -33.8551203729162
        ],
        [
          151.207382497044,
          -33.8550739215306
        ],
        [
          151.207589287023,
          -33.8550128320495
        ],
        [
          151.207620446008,
          -33.8550888286534
        ],
        [
          151.207628272004,
          -33.8551079129544
        ]
      ]
    ],
    [
      [
        [
          151.205319518151,
          -33.8564836954525
        ],
        [
          151.205286826154,
          -33.8564970876495
        ],
        [
          151.205264569157,
          -33.8564954113474
        ],
        [
          151.205228892175,
          -33.8564059687429
        ],
        [
          151.205307165166,
          -33.85638322715
        ],
        [
          151.205294679172,
          -33.8563536728485
        ],
        [
          151.205324780169,
          -33.8563445027512
        ],
        [
          151.205389159147,
          -33.8564347700585
        ],
        [
          151.205382786146,
          -33.8564524200581
        ],
        [
          151.205319518151,
          -33.8564836954525
        ]
      ]
    ],
    [
      [
        [
          151.207630907003,
          -33.8551143401547
        ],
        [
          151.207680998022,
          -33.854918118657
        ],
        [
          151.207200663072,
          -33.8550625373132
        ],
        [
          151.20709743412,
          -33.8548190487003
        ],
        [
          151.206826488248,
          -33.8541726291663
        ],
        [
          151.206642090334,
          -33.8537377416432
        ],
        [
          151.206639831334,
          -33.853737625143
        ],
        [
          151.206639285335,
          -33.8537365494429
        ],
        [
          151.206638702335,
          -33.8537354917428
        ],
        [
          151.206638083335,
          -33.8537344518428
        ],
        [
          151.206637438335,
          -33.8537334389427
        ],
        [
          151.206636757336,
          -33.8537324349426
        ],
        [
          151.206636040336,
          -33.8537314579425
        ],
        [
          151.206635305336,
          -33.8537305077424
        ],
        [
          151.206634534336,
          -33.8537295754424
        ],
        [
          151.206633727337,
          -33.8537286611423
        ],
        [
          151.206632902337,
          -33.8537277916422
        ],
        [
          151.206632051337,
          -33.8537269311421
        ],
        [
          151.206631163337,
          -33.853726106442
        ],
        [
          151.206630258337,
          -33.8537253086419
        ],
        [
          151.206629326338,
          -33.8537245377418
        ],
        [
          151.206628366338,
          -33.8537237937417
        ],
        [
          151.206627398338,
          -33.8537230855416
        ],
        [
          151.206626394338,
          -33.8537224132415
        ],
        [
          151.206625372339,
          -33.8537217588414
        ],
        [
          151.206624342339,
          -33.8537211493413
        ],
        [
          151.206623284339,
          -33.8537205666412
        ],
        [
          151.206622199339,
          -33.8537200109411
        ],
        [
          151.20662111534,
          -33.853719499941
        ],
        [
          151.20662000334,
          -33.8537190248408
        ],
        [
          151.20661887434,
          -33.8537185766407
        ],
        [
          151.20661773534,
          -33.8537181643406
        ],
        [
          151.20661658834,
          -33.8537177968405
        ],
        [
          151.206615422341,
          -33.8537174561404
        ],
        [
          151.206614257341,
          -33.8537171514403
        ],
        [
          151.206613083341,
          -33.8537168914401
        ],
        [
          151.206611891341,
          -33.8537166583401
        ],
        [
          151.206610698342,
          -33.8537164701399
        ],
        [
          151.206609497342,
          -33.8537163177398
        ],
        [
          151.206608287342,
          -33.8537162101397
        ],
        [
          151.206607086342,
          -33.8537161295396
        ],
        [
          151.206605876342,
          -33.8537160936395
        ],
        [
          151.206604666342,
          -33.8537160936393
        ],
        [
          151.206603456343,
          -33.8537161295392
        ],
        [
          151.206602254343,
          -33.8537162012391
        ],
        [
          151.206601053343,
          -33.853716317739
        ],
        [
          151.206599852343,
          -33.8537164701389
        ],
        [
          151.206598651343,
          -33.8537166494388
        ],
        [
          151.206597468343,
          -33.8537168735387
        ],
        [
          151.206596284344,
          -33.8537171424386
        ],
        [
          151.206595119344,
          -33.8537174382385
        ],
        [
          151.206593963344,
          -33.8537177788384
        ],
        [
          151.206592806344,
          -33.8537181464383
        ],
        [
          151.206591668344,
          -33.8537185587382
        ],
        [
          151.206590547344,
          -33.853718997938
        ],
        [
          151.20643799836,
          -33.8537627422241
        ],
        [
          151.20625118938,
          -33.853817592707
        ],
        [
          151.20625012238,
          -33.8538177451069
        ],
        [
          151.20624905638,
          -33.8538179334069
        ],
        [
          151.20624800738,
          -33.8538181664067
        ],
        [
          151.20624695838,
          -33.8538184354067
        ],
        [
          151.20624592738,
          -33.8538187491066
        ],
        [
          151.206244905381,
          -33.8538190897065
        ],
        [
          151.206243892381,
          -33.8538194752064
        ],
        [
          151.206242897381,
          -33.8538198965063
        ],
        [
          151.206241920381,
          -33.8538203537062
        ],
        [
          151.206240952381,
          -33.8538208467061
        ],
        [
          151.206240011381,
          -33.853821375506
        ],
        [
          151.206239097381,
          -33.853821931306
        ],
        [
          151.206238191381,
          -33.8538225319059
        ],
        [
          151.206237313381,
          -33.8538231594058
        ],
        [
          151.206236461381,
          -33.8538238138057
        ],
        [
          151.206235628381,
          -33.8538245040057
        ],
        [
          151.206234821381,
          -33.8538252211056
        ],
        [
          151.206234050381,
          -33.8538259651055
        ],
        [
          151.206233297381,
          -33.8538267450054
        ],
        [
          151.206232580381,
          -33.8538275517054
        ],
        [
          151.206231890381,
          -33.8538283764054
        ],
        [
          151.206231226381,
          -33.8538292370053
        ],
        [
          151.206230608381,
          -33.8538301154052
        ],
        [
          151.206230016381,
          -33.8538310118052
        ],
        [
          151.206229451381,
          -33.8538319441052
        ],
        [
          151.206228923381,
          -33.8538328853051
        ],
        [
          151.206228438381,
          -33.8538338444051
        ],
        [
          151.206227981381,
          -33.8538348215051
        ],
        [
          151.206227560381,
          -33.853835816505
        ],
        [
          151.206227175381,
          -33.853836820505
        ],
        [
          151.206226825381,
          -33.853837842405
        ],
        [
          151.20622652038,
          -33.853838882205
        ],
        [
          151.20622625138,
          -33.853839922005
        ],
        [
          151.20622601838,
          -33.8538409798049
        ],
        [
          151.20622583038,
          -33.8538420465049
        ],
        [
          151.20622567838,
          -33.8538431132049
        ],
        [
          151.20622556138,
          -33.853844188905
        ],
        [
          151.20622548938,
          -33.853845264605
        ],
        [
          151.20622545338,
          -33.853846340205
        ],
        [
          151.206225462379,
          -33.853847415905
        ],
        [
          151.206225507379,
          -33.853848491605
        ],
        [
          151.206225597379,
          -33.853849567305
        ],
        [
          151.206225722379,
          -33.853850634005
        ],
        [
          151.206225884379,
          -33.8538517007051
        ],
        [
          151.206226081379,
          -33.8538527674051
        ],
        [
          151.206226323378,
          -33.8538538162051
        ],
        [
          151.206226601378,
          -33.8538548650052
        ],
        [
          151.206226924378,
          -33.8538558958052
        ],
        [
          151.206225122378,
          -33.8538580382051
        ],
        [
          151.206596159204,
          -33.8547421735516
        ],
        [
          151.20679553611,
          -33.8552191468767
        ],
        [
          151.206795993105,
          -33.8552485756771
        ],
        [
          151.206799005079,
          -33.8554419466798
        ],
        [
          151.206584308087,
          -33.8556098867616
        ],
        [
          151.206345947096,
          -33.8557932180412
        ],
        [
          151.206243265124,
          -33.8556996520303
        ],
        [
          151.205587388433,
          -33.854135087848
        ],
        [
          151.205173880476,
          -33.8542582440103
        ],
        [
          151.205910020129,
          -33.8560130596026
        ],
        [
          151.205873339116,
          -33.8561523240009
        ],
        [
          151.20577423412,
          -33.8562296293925
        ],
        [
          151.205752030126,
          -33.8562095500901
        ],
        [
          151.205543662134,
          -33.8563694763723
        ],
        [
          151.205441840141,
          -33.8564223011633
        ],
        [
          151.205422505145,
          -33.8564182404615
        ],
        [
          151.20537643016,
          -33.8563539238562
        ],
        [
          151.205543492147,
          -33.8562706933711
        ],
        [
          151.205487897166,
          -33.8561929577648
        ],
        [
          151.205497175165,
          -33.8561883412656
        ],
        [
          151.205461731177,
          -33.8561387883616
        ],
        [
          151.205459750177,
          -33.8561393620614
        ],
        [
          151.204833545472,
          -33.8546496202829
        ],
        [
          151.204643365492,
          -33.8547052775655
        ],
        [
          151.204618669503,
          -33.8546464828624
        ],
        [
          151.204630358504,
          -33.8546273626633
        ],
        [
          151.204823613491,
          -33.8545181365803
        ],
        [
          151.204798280499,
          -33.8544869419775
        ],
        [
          151.204560296515,
          -33.8546214464566
        ],
        [
          151.204599863496,
          -33.8547156399615
        ],
        [
          151.204608172496,
          -33.8547132107623
        ],
        [
          151.204667308468,
          -33.8548538824697
        ],
        [
          151.204627579472,
          -33.8548656162661
        ],
        [
          151.204625957473,
          -33.8548616542659
        ],
        [
          151.204638363471,
          -33.854857916267
        ],
        [
          151.204548400514,
          -33.8546436591557
        ],
        [
          151.204527568518,
          -33.8546351703537
        ],
        [
          151.204175185555,
          -33.8547385160215
        ],
        [
          151.204163245554,
          -33.8547593931206
        ],
        [
          151.204182724545,
          -33.854806194023
        ],
        [
          151.204161461547,
          -33.8548122537211
        ],
        [
          151.204164670545,
          -33.8548199089215
        ],
        [
          151.203911465572,
          -33.8548931983984
        ],
        [
          151.203926229565,
          -33.8549283462002
        ],
        [
          151.203952592562,
          -33.8549206372027
        ],
        [
          151.204034585524,
          -33.8551157740129
        ],
        [
          151.20407028052,
          -33.8551053310162
        ],
        [
          151.204632025256,
          -33.8564423893866
        ],
        [
          151.204643159254,
          -33.8564422817877
        ],
        [
          151.204646359246,
          -33.8564994271887
        ],
        [
          151.204675528242,
          -33.8564991402915
        ],
        [
          151.204676836229,
          -33.8565916843928
        ],
        [
          151.2048767512,
          -33.8565897213118
        ],
        [
          151.204875873209,
          -33.8565276906109
        ],
        [
          151.204880408209,
          -33.8565235134113
        ],
        [
          151.204901796207,
          -33.8565176061132
        ],
        [
          151.204922620204,
          -33.8565118512151
        ],
        [
          151.20495257719,
          -33.8565840113189
        ],
        [
          151.204943999189,
          -33.8565986853183
        ],
        [
          151.204880301198,
          -33.8566069411123
        ],
        [
          151.204708148221,
          -33.8566188004961
        ],
        [
          151.20464806223,
          -33.8566194279904
        ],
        [
          151.204187788296,
          -33.8566242505467
        ],
        [
          151.204115046331,
          -33.8564396732375
        ],
        [
          151.204091784342,
          -33.8563840875345
        ],
        [
          151.204262754334,
          -33.8562659871493
        ],
        [
          151.203786355558,
          -33.8551279649895
        ],
        [
          151.20360903857,
          -33.8552309073739
        ],
        [
          151.203561870592,
          -33.855118543868
        ],
        [
          151.203144829636,
          -33.8552393245299
        ],
        [
          151.203346089541,
          -33.8557214521552
        ],
        [
          151.20362321041,
          -33.85638259949
        ],
        [
          151.203527645405,
          -33.8565201696826
        ],
        [
          151.203492990409,
          -33.8565340458795
        ],
        [
          151.203434456421,
          -33.8565047785736
        ],
        [
          151.203314105447,
          -33.8564437069614
        ],
        [
          151.203236531443,
          -33.8565508354554
        ],
        [
          151.202796516536,
          -33.8563379140108
        ],
        [
          151.202677519692,
          -33.8553214075866
        ],
        [
          151.202666852706,
          -33.8552334709844
        ],
        [
          151.202575007826,
          -33.8544543935658
        ],
        [
          151.208662051201,
          -33.8525697173203
        ],
        [
          151.208387942101,
          -33.8535954117073
        ],
        [
          151.208364340093,
          -33.8536786064061
        ],
        [
          151.208363201092,
          -33.8536826582061
        ],
        [
          151.208062297984,
          -33.8547940840916
        ],
        [
          151.207724660999,
          -33.8550434888627
        ],
        [
          151.207630907003,
          -33.8551143401547
        ]
      ]
    ]
  ]
}
```

**Comment by peterstace** [comment]
The bugfix for this has been included in the [v0.18.0](https://github.com/peterstace/simplefeatures/blob/master/CHANGELOG.md#v0180) release.
